exporter/context.go

```diff
@@ -499,6 +529,110 @@ func (ic *importContext) Run() error {
 	return nil
 }
 
+func isSupportedWsObject(obj workspace.ObjectStatus) bool {
+	return obj.ObjectType == workspace.Directory || obj.ObjectType == workspace.Notebook || obj.ObjectType == workspace.File
+}
+
+func (ic *importContext) generateResourceIdForWsObject(obj workspace.ObjectStatus) (string, string) {
+	var rtype string
+	switch obj.ObjectType {
+	case workspace.Directory:
+		rtype = "databricks_directory"
+	case workspace.File:
+		rtype = "databricks_workspace_file"
+	case workspace.Notebook:
+		rtype = "databricks_notebook"
+	default:
+		log.Printf("[WARN] Unsupported WS object type: %s in obj %v", obj.ObjectType, obj)
```

Comment:
Should we fail here?

Reply:
it's too late to fail - it's better to don't some missing code that could be added manually, than fail multi-hour work. Although, this block is just defensive piece - because we have filters earlier we don't reach it.